packages/devtools_app/lib/src/ui/utils.dart

```diff
@@ -134,6 +134,19 @@ double calculateTextSpanHeight(TextSpan span) {
   return textPainter.height;
 }
 
+TextSpan findLongestTextSpan(List<TextSpan> spans) {
```

Comment:
now that this is easier to test, can you add tests for this in utils_test.dart?

Reply:
Good idea, added tests